app/_landing_pages/gateway/rate-limiting.yaml

```diff
@@ -0,0 +1,95 @@
+metadata:
+  title: Gateway Rate Limiting
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "Rate Limiting with Kong Gateway"
+              align: center
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "Capabilities"
+              blocks:
+                - type: text
+                  text: |
+                    Kong Gateway allows you to limit how many HTTP requests can be made in a given period.
+
+                    Rate limits can be applied to a [service](/gateway/entities/service/) or individual [routes](/gateway/entities/route/) to protect the upstream API, or to a [consumer](/gateway/entities/consumer/) to restrict how many times a user can call the API in the specified period.
+
+                    In addition to the OSS Rate Limiting plugin, Kong offer an Enterprise rate limiting plugin with additional capabilities.
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "Open Source Features"
```

Comment:
I think this formatting works, but I think for a feature comparison, a feature support table might work better to get info at a glance.

Reply:
I'd say let's ship this, we can ask the design team for feedback

Reply:
This definitely isn't the end result. The content is indicative, but it needs a ton of design work / layout fixes